src/components/FoundationGrid/index.stories.js

```diff
@@ -2,12 +2,12 @@ import React from 'react'
 import { storiesOf } from '@storybook/react'
 import { withInfo } from '@storybook/addon-info'
 
-import GridExample from '../GridExample'
+import FoundationGrid from '../FoundationGrid'
 
-storiesOf('foundation|GridExample', module)
+storiesOf('foundation|Grid', module)
   .add(
     'default',
     withInfo()(() =>
-      <GridExample />,
+      <FoundationGrid />,
```

Comment:
Check my PR, we can actually just use HTML here.  Having a component for this hides the markup in the documenation.